crates/core/src/factories/core_publish.rs

```diff
@@ -315,8 +315,8 @@ impl CorePublish {
             // Add incoming op ids to the fetch queue to let that retrieve the op data
             if let Err(err) = fetch.request_ops(op_ids.clone(), peer).await {
                 tracing::warn!(
-                        "could not insert publish ops request into fetch queue: {err}"
-                    );
+                    "could not insert publish ops request into fetch queue: {err}"
```

Comment:
I like this.

Reply:
I think this was actually over-indented before, it's displaying like

```
if let Err(err) = fetch.request_ops(op_ids.clone(), peer).await {
      tracing::warn!(
          "could not insert publish ops request into fetch queue: {err}"
      );
};
```

For me locally

Reply:
So the closing `);` is aligned with the call and the argument is indented. That's more in line with the formatting in other languages